OlinkAnalyze/vignettes/bridging_E3072toEHT.Rmd

```diff
@@ -0,0 +1,682 @@
+## After bridging
+
+### Keep the data following BridgingRecommendation
+npx_after_br_reco <- npx_br_data |>
+  filter(BridgingRecommendation != "Not Bridgeable") |>
+  mutate(NPX = case_when(
+    BridgingRecommendation == "MedianCenteredNPX" ~ MedianCenteredNPX,
+    BridgingRecommendation == "QSNormalizedNPX" ~ QSNormalizedNPX,
+    .default = NPX)) |>
+  filter(!stringr::str_detect(Assay, "Inc|Amp|Ext"))
+
+### Generate unique SampleIDs
+npx_after_br_final <- npx_after_br_reco |>
+  dplyr::mutate(Type = ifelse(SampleID %in% overlapping_samples, 
+                              paste(Project, "Bridge"),
+                              paste(Project, "Sample"))) |>
+  dplyr:::mutate(SampleID = paste0(Project, PlateID, SampleID))
+  
+### PCA plot of the data from SCs
+  npx_after_br_final |> filter(stringr::str_detect(SampleID, "CONTROL_SAMPLE")) |> 
+    OlinkAnalyze::olink_pca_plot(color_g     = "Type")
+  
+### PCA plot of the data from bridging samples
+  npx_after_br_final |> filter(stringr::str_detect(Type, "Bridge")) |> 
+    OlinkAnalyze::olink_pca_plot(color_g     = "Type")
+```
+
+## Exporting Normalized Data
+
+The normalized Explore 3072 data can be exported using
+`arrow::write_parquet()` to create a long format Olink Explore file.
+
+```{r, eval=FALSE}
+df <- npx_br_data |>
+    dplyr::filter(Project == "Explore_3072_NAME") |>
+    arrow::as_arrow_table()
+
+df$metadata$FileVersion <- "NA"
+df$metadata$ExploreVersion <- "NA"
+df$metadata$ProjectName <- "NA"
+df$metadata$SampleMatrix <- "NA"
+df$metadata$DataFileType <- "Olink Analyze Export File"
+df$metadata$ProductType <- "Explore3072"
+df$metadata$Product <- "Explore3072"
+arrow::write_parquet(x = df, sink = "path_to_output.parquet")
+```
+
+## FAQs
+
+### Overlapping Assays within products
+
+Both the Explore 3072 and Explore HT products contain assays that appear
+multiple times in the product, known as overlapping assays or
+correlation assays. In Explore 3072, these present as overlapping assays
+across panels. In Explore HT, these are overlapping assays across
+blocks. These assays are included for QC purposes and allow users to
+evaluate data performance across panels in Explore 3072 and across
+blocks in Explore HT. IL6, IL8 (CXCL8), and TNF are included in the
+Cardiometabolic, Oncology, Neurology and Inflammation panels, while
+IDO1, LMOD1, and SCRIB are included in the Cardiometabolic II, Oncology
+II, Neurology II and Inflammation II panels. Each correlation assay is
+measured four times in an Olink Explore 3072 run. In Explore HT, GBP1
+and MAPK1 serve as correlation assays and are measured three times in a
+run.
+
+Within each product, the assays contain unique OlinkID values for each
+of their corresponding panels and blocks in Explore 3072 and Explore HT,
+respectively. It should be noted that the overlapping assays themselves
+differ between each product and that these assays differ from the
+overlapping assays *across* products.
+
+### Downstream Analysis
+
```

Comment:
TBD - will we have the olink_normalization_bridge_product() wrapper ready for initial launch? If not, should we just refer to olink_normalization()? Will leave as-is for now.